### DISCUSSION TITLE
I am getting this error when deploying azure jumpstart hcibox.

### DISCUSSION THREAD

**Discussion by gabrielluizbh** [microsoft/azure_arc#2134] [Q&A]
![image](https://github.com/microsoft/azure_arc/assets/31531703/ab5691c2-cf2c-40fe-a899-a9110177b0fe)

**Comment by gabrielluizbh** [comment]
![image](https://github.com/microsoft/azure_arc/assets/31531703/0a989d61-e8be-4ba3-8433-3f5591ebc252)

**Reply by dkirby-ms** [reply]
Can you try running Connect-AzAccount manually before trying to run azd up?

**Comment by gabrielluizbh** [comment]
I tried and still got an error!

![image](https://github.com/microsoft/azure_arc/assets/31531703/d41eabdb-9809-4ed1-bacd-b07d842d99eb)

**Comment by gabrielluizbh** [comment]
I'm sorry, but I found the documentation presented very difficult to understand. 

https://azurearcjumpstart.io/azure_jumpstart_hcibox/#start-post-deployment-automation

**Reply by likamrat** [reply]
Gabriel, can you please be more specific what exactly is unclear and difficult to understand? Can you also please try what Dale asked?

**Reply by gabrielluizbh** [reply]
Already tried!

**Reply by likamrat** [reply]
Gabriel, can you please be more specific about what exactly is unclear and difficult to understand? 
What was the output of your attempt following Dale's suggestion?

**Reply by gabrielluizbh** [reply]
It was demonstrated in the image.

**Reply by gabrielluizbh** [reply]
https://azurearcjumpstart.io/azure_jumpstart_hcibox/#start-post-deployment-automation

First suggestion for improvement in documentation, separate what will be deployed Overview from the technical deployment documentation.

This way it looks better visually.

**Comment by dkirby-ms** [comment]
You can try deleting the .azure folder and re-running azd init. Make sure when you select your subscription when running azd up for the first time that its the correct subscription.

**Reply by gabrielluizbh** [reply]
I'm writing my doubts and difficulties regarding the deployment documentation. I'll post it here soon.
